static/.htaccess

```diff
@@ -317,15 +297,16 @@ RewriteRule ^contact/?$ /docs/reference/contact/ [R=301,L]
 RewriteRule ^docs/components/connectors/custom-built-connectors/update-guide/(.*)$ /docs/components/connectors/custom-built-connectors/connector-sdk/ [R=301,L]
 
 # Remove outdated message correlation guide
-RewriteRule ^docs/guides/message-correlation/?$ /docs/guides/$1 [R=301,L]
-RewriteRule ^docs/8.5/guides/message-correlation/?$ /docs/guides/$1 [R=301,L]
+RewriteRule ^docs/guides/message-correlation/?$ /docs/guides/ [R=301,L]
+RewriteRule ^docs/8.6/guides/message-correlation/?$ /docs/guides/ [R=301,L]
```

Comment:
This one wasn't in the left side, but I added it because it's strange for us to jump from 8.5 to 8.7, and I think it was probably intended to exist for 8.6 also.